cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -111,8 +111,8 @@ data TraceOptions = TraceOptions
   , traceDnsResolver     :: !Bool
   }
 
-type ConsensusTraceOptions = Consensus.Tracers' () ()    (Const Bool)
-type ProtocolTraceOptions  = ProtocolTracers'   () () () (Const Bool)
+type ConsensusTraceOptions = Consensus.Tracers' () ()    () (Const Bool)
```

Comment:
Any reason for the spaces here?

Reply:
I thought people here liked this kind of alignment. I don't really care either way.